lib/lipseys/order.rb

```diff
@@ -7,7 +7,7 @@ class Order < Base
 
     SUBMIT_TO_STORE_ATTRS = {
       permitted: %i( po_number disable_email items item_no quantity note ).freeze,
-      required:  %i( items item_no quantity ).freeze
+      required:  %i( po_number items item_no quantity ).freeze
```

Comment:
@jeffdill2 I think this is a breaking change, if anyone was not currently sending that variable for orders. It was required before, by lipseys, but we're actually checking for it now. I think this might be the only breaking change though.

Reply:
So Lipseys was already requiring that before – as in if you didn't send it, it would fail?

Reply:
Correct. So if it weren't being provided before, the order would be rejected by Lipseys. So it's broken if you weren't doing sending it, and if you aren't sending it after we start requiring it it's still broken. So is the change really breaking anything 🤔🤯

Reply:
Yeah, so in that case I'd say it's not really a breaking change.

Reply:
Word. I think that would be the only breaking change, so I'm thinking `6.2.0`?

Reply:
![](https://h2.gifposter.com/gifs/movie/okay,-agreeing,-approval,-smile,-Eddie-Murphy,-I-Spy.gif)

Reply:
💥